tests/helpers.py

```diff
@@ -21,3 +26,69 @@ def name(self):
     def filename(self):
         # weird flask property
         return self._filename
+
+
+@contextmanager
+def assert_log_entry(modules, message, count=1):
+    """A context manager that can be used to wrap a block of code under test to assert that a specific message is
+    logged. Works well with the AnyStringMatching malleable above to assert a message meeting a given format.
+
+    This is designed to most easily assert a single kind of message being logged one or more times, but if you
+    assign the context manager to a variable you can access the `records` attribute to see all log records and make
+    further inspections.
+
+    Example:
+        with AssertLogEntry(message=AnyStringMatching('^My unknown string with a random word \w+$', count=2)) as logs:
```

Comment:
Have we changed our name about the name/casing of this thing since the docblock was written, or am I just confused?

Reply:
Ah yeah good spot. Originally the `assert_log_entry` thing was a class. I need to update the docstring.